feat(apple): Enable AppHangs by default (#6064)
getsentry/sentry-docs: src/platforms/apple/common/configuration/app-hangs.mdx

Because the app hang detection integration uses SentryCrashIntegration to capture the stack trace when creating app hang events, if SentryCrashIntegration is disabled, the integration won’t work.

> Since 8.0.0 this feature is enable dy default. To disable this feature:

Reviewer: ```suggestion
Since 8.0.0, this feature has been enabled by default. To disable it:
```
